vi:
  devise:
    omniauth_callbacks:
      failure: "Không thể xác thực từ %{kind} do nguyên nhân \"%{reason}\"."
      success: "Xác thực thành công với tài khoản %{kind}."
    failure:
      already_authenticated: "Xác thực thành công"
      unauthenticated: "Bạn cần đăng nhập để tiếp tục"
    sessions:
      signed_in: "Đăng nhập thành công."
      signed_out: "Đăng xuất thành công."
      already_signed_out: "Đăng xuất thành công."
      login_btn: "Đăng nhập"
      logout_btn: "Đăng xuất"